### ISSUE TITLE
among us opens differently

### ISSUE THREAD

**Issue by khead324** [ottomated/CrewLink#622]
among us opens differently or something bc it says open game on crew link but i have the game open (there was a update today in among us)

**Comment by haybako55** [comment]
when you open it up does it just stuck on the screen that says menu on it?

**Comment by DirtyaiDAN** [comment]
Doing the same for me and my friends. When we open Crewlink it just says "menu" and never changes even when in a lobby. I'm assuming it probably just needs an update since among us updated?

**Comment by khead324** [comment]
yeah it just sits there i have tried offical servers and unoffical

**Comment by MatadorProBr** [comment]
Look the issue in [#623](https://github.com/ottomated/CrewLink/issues/623#issuecomment-793888862) that solves this problem.
